entities/global.ent

```diff
@@ -367,6 +367,7 @@
 <!ENTITY url.php.release4.2.0 "https://www.php.net/releases/4_2_0.php">
 <!ENTITY url.php.svn "https://svn.php.net/">
 <!ENTITY url.php.git.phpini "https://github.com/php/php-src/blob/master/php.ini-production">
+<!ENTITY url.php.git.src "https://github.com/php/php-src/blob/master">
```

Comment:
Maybe we should replace `&url.php.git.phpini;` with `&url.php.git.src;/php.ini-production`. Would be a bit ugly for the translations, though, since they won't build until they are updated.

Reply:
Consider waiting awhile before implementing such things; so imho wait on the url.php.git.phpini integration.

Reply:
Upon further review, I request the following change:

Modify the entity name as it’s a generic name but is a 404 without adding a path. As a similar example, we have this in `global.ent`:
```
  <!-- linking to specific pecl packages is done like so: &url.pecl.package;package_name -->
  <!ENTITY url.pecl.package.get "https://pecl.php.net/get/">
```
This PR feels similar as it’s meant to view specific parts of the master's source. As such, a name like `url.php.git.src.master.view` makes sense along with a comment similar to the above. This way we know it views a source file from php master in git.

Reply:
let me do a summary for what I should do next (@philip @cmb69 pls correct me for any mistaken). 
update my patch as below and resubmit this PR:
<!-- linking to specific source file is done like so: &url.php.git.src.master.view;file_path -->
<!ENTITY url.php.git.src.master.view "https://github.com/php/php-src/blob/master/">

Reply:
As for 'php.ini-production' link, we can leave it alone there, or I can fix it by using our new url.php.git.src.master.view in another PR?

Reply:
Yep, sounds good; and I vote to ignore the `php.ini-production` link for now.